CIME/scripts/create_newcase.py

```diff
@@ -231,12 +231,21 @@ def parse_command_line(args, cimeroot, description):
         "--input-dir",
         help="Use a non-default location for input files. This will change the xml value of DIN_LOC_ROOT.",
     )
-    parser.add_argument(
-        "--driver",
-        default=get_cime_default_driver(),
-        choices=("mct", "nuopc", "moab"),
-        help=argparse.SUPPRESS,
-    )
+    if model == "cesm":
+        drv_choices = ("mct", "nuopc")
+    elif model == "e3sm":
+        drv_choices = ("mct", "moab")
```

Comment:
I think that you should put the argparse.SUPPRESS option back for e3sm.
UFS only uses nuopc so they don't need this option.

Reply:
OK, I'll do that.

Reply:
OK, did that.